worker/utils/utils.js

```diff
@@ -49,6 +51,27 @@ module.exports = {
     })
   },
 
+  async encryptJob(string1, string2){
```

Comment:
`encrypt` is inaccurate nomenclature: encryption is reversible with the key, but the purpose of password hashing is that this procesws should be irreversible.

I would go with `pwhash` or something like that.